ci/azure-pipelines-pte-daily.yml

```diff
@@ -39,12 +39,15 @@ jobs:
       mkdir -p logs
       ls logs
       cp tools/PTE/CITest/Logs/*.* logs
+    condition: always()
     displayName: Copy Logs
   - task: PublishBuildArtifacts@1
+    condition: or(succeeded(), failed())   # publish either way
     inputs:
       pathtoPublish: 'logs'
```

Comment:
Copy Logs task can be removed and PublishBuildArtifacts@1 can be used to publish Logs directly without copying like pathtoPublish: 'tools/PTE/CITest/Logs'